lib/SpotifyAccess.js

```diff
@@ -0,0 +1,23 @@
+require("dotenv").config();
```

Comment:
This should only be included once in the server.js 
https://www.npmjs.com/package/dotenv#usage